ci/jenkins/pipelines/prs/helpers/pr_manager/pr_checks.py

```diff
@@ -24,6 +27,51 @@ def check_pr_from_fork(self, pr_number):
 
         print(f'PR-{pr_number} is from a fork.')
 
+    def check_pr_manifest_change(self):
+        """
+        Checks if the given PR has updated any Addon and that its respective manifest
+        has been bumped.
+
+        The Regular expression will match:
+        1: + or - symbol
+        2: Addon Name
+        3: Version
+        4: Manifest
+
+        :return:
+        """
+
+        added_changes = []
+        removed_changes = []
+        p = re.compile('(-|\+)([A-Za-z]+):&AddonVersion{"(.+)",([0-9]+)},')
+
+        cmd = f'git diff --no-ext-diff --unified=0 master -- {WORKSPACE}/internal/pkg/skuba/kubernetes/versions.go | grep "AddonVersion"'
```

Comment:
Jenkins do a pre-merge of the PR into the master branch, so this returns empty always if run on jenkins:

```
commit fb7b23e629f238a48b7ee3778c4d2fb1453350e1 (HEAD -> master)
Merge: 09b42604 93cc2b65
Author: jenkins worker <worker@jenkins.ci>
Date:   Fri Nov 27 09:21:42 2020 +0000

    Merge commit '93cc2b650b930e8c17cdf03edb81f90b3d6ce708' into HEAD
```

You can thank https://github.com/SUSE/skuba/blob/master/ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile#L192 for that :D

```suggestion
        cmd = f'git diff --no-ext-diff --unified=0 HEAD~1 -- {WORKSPACE}/internal/pkg/skuba/kubernetes/versions.go | grep "AddonVersion"'
```

Reply:
This works as expected on jenkins:

```
jenkins@caasp-team-43-jenkins-worker:~/workspace/caasp-jobs_pr_test_PR-1449> git diff --no-ext-diff --unified=0 HEAD~1 -- /home/jenkins/workspace/caasp-jobs_pr_test_PR-1449/internal/pkg/skuba/kubernetes/versions.go|grep "AddonVersion"
-				Gangway:       &AddonVersion{"3.1.0-rev5", 4520},
+				Gangway:       &AddonVersion{"3.1.0-rev6", 4520},
-				Kucero:        &AddonVersion{"1.3.0", 0},
+				Kucero:        &AddonVersion{"1.3.1", 0},
-				MetricsServer: &AddonVersion{"0.3.6", 1},
+				MetricsServer: &AddonVersion{"0.3.7", 1},
```

Wont work on local though....